scripts/docker-entrypoint.sh

```diff
@@ -289,6 +293,46 @@ else
   echo "No web.xml was specified in ${web_xml}. Using defaults."
 fi
 
+#
+# Copying mounted catalina.properties file to conf
+#
+if [ -e "${catalina_properties}" ]; then
+  echo "Loading catalina.properties from ${catalina_properties}...";
+  cp "${catalina_properties}" "${CATALINA_HOME}/conf/"
+else
+  echo "No catalina.properties was specified in ${catalina_properties}. Using defaults."
+fi
+
+#
+# Copying mounted prbootstrap properties file to webapps/prweb/WEB-INF/classes
+#
+if [ -e "$prbootstrap_properties" ]; then
+  echo "Loading prbootstrap.properties from ${prbootstrap_properties}...";
+  cp "$prbootstrap_properties" ${PEGA_DEPLOYMENT_DIR}/WEB-INF/classes/
```

Comment:
Just curious the other cp commands has second argument in double quotes. Should this be also changed? Are we able to test it?

Reply:
Took reference from prlog4j2 and prconfig, where files are getting copied into WEB-INF/classes, where quotes are not mentioned.

Unittests were added.

Reply:
For [web.xml](https://github.com/pegasystems/docker-pega-web-ready/blob/master/scripts/docker-entrypoint.sh#L287) it is added that's where I checked. If its tested we are good.

Reply:
Tested in local as well.  It is working as expected.